### PR TITLE
Feat database migration

### PR DESCRIPTION
**Pull request by RuslanKamenev**

### CODE REVIEW

File: backend/database/migrations/2012_02_07_130822_create_roles.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateRoles extends Migration
```
Comment (RuslanKamenev) [inline backend/database/migrations/2012_02_07_130822_create_roles.php:7]: Rename migrations

File: backend/database/migrations/2022_02_07_144229_create_locations.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateLocations extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
```
Comment (RuslanKamenev) [inline backend/database/migrations/2022_02_07_144229_create_locations.php:13]: Fix table

Comment (RuslanKamenev) [reply backend/database/migrations/2022_02_07_144229_create_locations.php:13]: Add location table

File: backend/database/migrations/2022_02_07_144119_create_event_types.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('event_types', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->text('description');
+            $table->unsignedBigInteger('user_id');
```
Comment (RuslanKamenev) [inline backend/database/migrations/2022_02_07_144119_create_event_types.php:20]: Add events table

**Comment by RuslanKamenev** [comment]
Removed comments in files.
Migrations renamed (added "_table" to all files).
Romoved default value in "role" column in "create_roles_table".
Changed column type to JSON (column "settings" in "user_settings_table").
Fixed column names in "create_locations_table".

File: backend/database/migrations/2022_02_07_144229_create_locations_table.php [hunk 4]
```diff
@@ -4,20 +4,15 @@
...
-        Schema::create('event_types', function (Blueprint $table) {
+        Schema::create('locations', function (Blueprint $table) {
             $table->id();
+            $table->unsignedBigInteger('user_id');
             $table->string('name');
             $table->text('description');
-            $table->unsignedBigInteger('user_id');
             $table->float('longitude');
             $table->float('latitude');
             $table->tinyInteger('is_hidden');
```
Comment (RomanMahola) [inline backend/database/migrations/2022_02_07_144229_create_locations_table.php:18]: https://i.imgur.com/82J6KSQ.png
No timestamps

File: backend/database/migrations/2022_02_07_135420_create_user_interests_table.php [hunk 5]
```diff
@@ -4,13 +4,8 @@
...
+class CreateUserInterestsTable extends Migration
 {
-    /**
-     * Run the migrations.
-     *
-     * @return void
-     */
     public function up()
     {
         Schema::create('user_interests', function (Blueprint $table) {
```
Comment (RomanMahola) [inline backend/database/migrations/2022_02_07_135420_create_user_interests_table.php:11]: No Timestamps

**Pull request merged**